src/Resources/Resources.Test/ScenarioTests/PolicyTests.ps1

```diff
@@ -881,13 +908,14 @@ $policyAssignmentNotFound = 'PolicyAssignmentNotFound : '
 $policySetDefinitionNotFound = 'PolicySetDefinitionNotFound : '
 $policyDefinitionNotFound = 'PolicyDefinitionNotFound : '
 $invalidRequestContent = 'InvalidRequestContent : The request content was invalid and could not be deserialized: '
-$missingSubscription = 'MissingSubscription : The request did not have a provided subscription. All requests must have an associated subscription Id.'
+#$missingSubscription = 'MissingSubscription : The request did not have a provided subscription. All requests must have an associated subscription Id.'
```

Comment:
@mentat9 please delete this line if it's going to be commented-out

Reply:
Done.